### PR TITLE
PR: Deactivate class/function dropdown by default

### PR DESCRIPTION
**Pull request by rlaverde**
Fixes: #4948

### CODE REVIEW

File: spyder/plugins/editor.py [hunk 1]
```diff
@@ -995,6 +995,13 @@ def get_plugin_actions(self):
         showindentguides_action = create_action(self, _("Show indent guides."),
                                                toggled=self.toggle_show_indent_guides)
         showindentguides_action.setChecked(CONF.get('editor', 'indent_guides'))
+
+        show_classfunc_dropdown_action = create_action(
+                self, _("Show class function dropdown."),
```
Comment (ccordoba12) [inline spyder/plugins/editor.py]: Please change this to

    Show class and methods dropdown

**Comment by ccordoba12** [comment]
I left a minor comment, then I'll merge.

File: spyder/plugins/editor.py (hunk 1 above)
Comment (rlaverde) [reply spyder/plugins/editor.py]: In the preferences says: `Show Class/Function Dropdown`, should I change it too?

Comment (ccordoba12) [reply spyder/plugins/editor.py]: I wasn't aware of that. Then let's leave it (in both places) as

    Show selector for classes and functions

**Pull request merged**